jdaviz/core/template_mixin.py

```diff
@@ -67,8 +67,9 @@
 GLUE_STATES_WITH_HELPERS = ('size_att', 'cmap_att')
 
 # this histogram viewer (along with other viewers) are not in the glue viewer-registry by default
-# but may be added in the future.  If it is not in the registry, we'll add it now.  If/once the
-# min-pin of glue-jupyter includes this in the registry, we can safely remove this block.
+# but may be added in the future.  If it is not in the registry, we'll add it now.
+# Once glue-jupyter with https://github.com/glue-viz/glue-jupyter/pull/402 is pinned,
```

Comment:
If you know the version to pin to, please also document it here. Or open issue so we don't forget. Thanks!

Reply:
I do not know the version yet (likely the next minor release, which would be 0.20.0, but can't be sure... I guess anything _after_ 0.19.0 since its now merged) and forgetting won't hurt - we can remove this at any point after its sure to be available upstream.